bin/g

```diff
@@ -588,15 +611,8 @@ install() {
 
   dir="$GO_VERSIONS_DIR/$version"
 
-  if [ -d "$dir" ]; then
-    if [ "$ACTIVATE" = true ]; then
-      activate "$version"
-    fi
-    exit
-  fi
-
   echo
-  log_info "installing" "$version"
+  log_info "downloading" "$version"
```

Comment:
Let's re-rephrase to "selected"; this log has been important mainly to denote what version was resolved when using the `latest` keyword.